### PR TITLE
Vb 3544 - add a new endpoint to add notifications that are not SQS based

### PR DESCRIPTION
**Pull request by dhirajshettymoj**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsprisonvisitstestinghelperapi/controller/TestingDBApiBHelperController.kt [hunk 1]
```diff
@@ -81,4 +85,32 @@ class TestingDBApiHelperController {
     val recordsDeleted = dBService.deleteVisitNotificationEvents(reference)
     return ResponseEntity(recordsDeleted, OK)
   }
+
+  @PreAuthorize("hasAnyRole('VISIT_SCHEDULER','VISIT_SCHEDULER_CONFIG','TEST_VISIT_SCHEDULER')")
```
[the remark this answers is not in the document]
Comment (AledWEvans) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsprisonvisitstestinghelperapi/controller/TestingDBApiBHelperController.kt:89]: could you all prevent vent this from happening in prod, something like

@Profile("!prod")

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsprisonvisitstestinghelperapi/dto/enums/DBNotificationEventType.kt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package uk.gov.justice.digital.hmpps.hmppsprisonvisitstestinghelperapi.dto.enums
+
+enum class DBNotificationEventType {
```
Comment (AledWEvans) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsprisonvisitstestinghelperapi/dto/enums/DBNotificationEventType.kt:3]: it's more of a Test event

Comment (dhirajshettymoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsprisonvisitstestinghelperapi/dto/enums/DBNotificationEventType.kt]: Updated enum name to TestDBNotificationEventTypes

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsprisonvisitstestinghelperapi/controller/TestingDBApiBHelperController.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhirajshettymoj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsprisonvisitstestinghelperapi/controller/TestingDBApiBHelperController.kt:89]: Raised - https://dsdmoj.atlassian.net/browse/VB-3548 for role restriction.

**Review by AledWEvans — APPROVED** [review]
LGTM

**Pull request merged**